feature/home/src/main/java/com/epmedu/animeal/home/presentation/ui/ComposableMapboxMap.kt

```diff
@@ -0,0 +1,25 @@
+package com.epmedu.animeal.home.presentation.ui
```

Comment:
This file looks like a consequence of rebasing onto develop branch. Please remove it